Makefile

```diff
@@ -18,3 +18,19 @@ IMAGE_REGISTRY :=registry.ci.openshift.org
 $(call build-image,ocp-must-gather,$(IMAGE_REGISTRY)/ocp/4.17:ocp-must-gather, ./Dockerfile.ocp,.)
 
 $(call verify-golang-versions,Dockerfile.ocp)
+
+# Allow users to pass in an authentication file when building with podman
+# An absolute path needs to be passed in here
+# This file must container credentials for registry.ci.openshift.org
+AUTH_FILE ?=
+
+# Target for building the image using Podman
+.PHONY: podman-image
+podman-image:
```

Comment:
https://github.com/openshift/build-machinery-go/blob/master/make/targets/openshift/images.mk#L18 has a similar structure here. Other repositories might benefit from this change if `make/targets/openshift/images.mk` gets extended. E.g. `IMAGE_BUILD_BUILDER ?= imagebuilder` and
```
image-$(1): ensure-imagebuilder
	$(strip \
		$(IMAGE_BUILD_BUILDER) \
		$(IMAGE_BUILD_DEFAULT_FLAGS) \
		-t $(2)
		-f $(3) \
		$(IMAGE_BUILD_EXTRA_FLAGS) \
		$(4) \
	)
.PHONY: image-$(1)
```
With https://github.com/openshift/build-machinery-go/blob/master/make/targets/openshift/imagebuilder.mk updated to skip the check for `imagebuilder` binary when IMAGE_BUILD_BUILDER != "imagebuilder". Would you be fine exploring this approach?

Reply:
@ingvagabund I have created https://github.com/openshift/build-machinery-go/pull/99 following your suggestion, will vendor the new changes here and update once that is merged.